pkg/vault/contracts/BalancerContractRegistry.sol

```diff
@@ -293,4 +300,9 @@ contract BalancerContractRegistry is IBalancerContractRegistry, SingletonAuthent
     function _getContractId(string memory contractName) internal pure returns (bytes32) {
         return keccak256(abi.encodePacked(contractName));
     }
+
```

Comment:
Would put this above `_getContractId` so that the most general internal one stays on the bottom.